_posts/2024-02-23-gsoc2024.md

```diff
@@ -0,0 +1,171 @@
+
+In short:
+Agree to be bound by our [Code of Conduct];
+Familiarize yourself with MDAnalysis;
+Submit a short pre-proposal — 
+**If you have been invited to proceed with your application (based on your pre-prosal)** have a pull request merged in MDAnalysis.
+
+
+### Code of Conduct
+
+The MDAnalysis community values diversity and is committed to providing a
+productive, harassment-free environment to every member. Our [Code of Conduct]
+explains the values that we as a community uphold. *Every member (and every
+GSoC Contributor) agrees to follow the [Code of Conduct]*.
+
+
+### Familiarize yourself with MDAnalysis
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+Watch the MDAnalysis Trailer
+Complete the Quick Start Guide
+
+#### Watch the MDAnalysis Trailer
+
+The [MDAnalysis Trailer](https://www.youtube.com/watch?v=uMAfvwFkD3o) on YouTube
+is a one minute introduction to MDAnalysis.
+
+<div class="js-video">
+	<iframe src="https://www.youtube.com/embed/uMAfvwFkD3o" frameborder="0"
+	allowfullscreen class="video"></iframe>
+</div>
+
+There are also additional [introductory videos](https://www.mdanalysis.org/pages/learning_MDAnalysis/#introductory) available to give you an idea of what problems MDAnalysis is solving.
+
+#### Complete the Quick Start Guide
+
+Start by [installing] the MDAnalysis package. We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+### Submit a pre-proposal
+
+We ask you to submit a short *pre-proposal* so that we can evaluate if you are a good fit for work on MDAnalysis. In this pre-proposal you tell us what project you’d like to work on, what you consider the major challenges, and how you plan to solve the problem. MDAnalysis developers will review your proposal and then either invite you to continue with your application and work on an issue to get involved in the code development or tell you that we will not consider your application. A rejection is not a judgment on you or your abilities but an assessment of how well you fit the specific requirements of MDAnalysis as a scientific software package — we much rather you spend your energies on contributing to another exciting GSoC project than being rejected later by MDAnalysis.
+
+**We require GSoC applicants to submit a pre-proposal** that will determine who is then invited to attempt a pull request (see below). If you are invited to attempt a pull request and ultimately submit a full application, the pre-proposal will help you gather some of the information you will need to include. 
+
+Submit your pre-proposal via [this submission form](https://forms.gle/wbKZDWmzAk1GVMgR6) **as soon as possible, but no later than March 15, 2024**. You should prepare the following information for your pre-proposal:
+* Email Address
+* GitHub Handle
+* Real Name (optional)
```

Comment:
```suggestion
* Real name (optional)
```
Why optional?

Reply:
I had written "optional" originally because I don't think that GSOC requires clear names... I might be wrong. In any case, it's not clear to me that we do need real names and then I think it's fine to leave it up to applicant.